### PR TITLE
update the getting setup instructions

### PR DESCRIPTION
**Pull request by clarkbw**
Just a quarterly review of our setup instructions.

### CODE REVIEW

File: docs/getting-setup.md [hunk 1]
```diff
@@ -73,11 +73,11 @@ This command starts a [development server][dev-server].
 
 ### Step 5. Open the Debugger
 
-Go to `localhost:8000` in any browser to view the Debugger. If everything worked successfully, you should see this [screenshot](https://cloud.githubusercontent.com/assets/254562/20439428/7498808a-ad89-11e6-895d-d6db320c5009.png)
+Go to http://localhost:8000 in any browser to view the Debugger. If everything worked successfully, you should see something like this [screenshot](https://cloud.githubusercontent.com/assets/2134/22162697/913777b2-df04-11e6-9150-f6ad676c31ef.png) :sweat_smile:
```
Comment (jasonLaster) [inline docs/getting-setup.md]: Lets move this to a url at the bottom of the page

File: docs/getting-setup.md [hunk 2]
```diff
@@ -126,9 +126,9 @@ Close firefox and re-open it with the `firefox-bin` command.
 
 **Firefox GCLI**
 
-* open Firefox
-* *shift-F2* Open GCLI
-* *listen 6080* - start listening on 6080
+* Open Firefox
+* *<kbd>shift</kbd>+<kbd>F2</kbd>* Open GCLI
```
Comment (jasonLaster) [inline docs/getting-setup.md]: this is the coolest thing!

File: docs/getting-setup.md (hunk 1 above)
Comment (clarkbw) [inline docs/getting-setup.md]: I wasn't sure about the screenshots, there's no change to use them anywhere else.  I can move them if you feel strongly, I just wasn't sure.

Comment (jasonLaster) [inline docs/getting-setup.md]: meh, i don't feel strongly. My main reason is i like reading the prose without the long urls.

Comment (jasonLaster) [inline docs/getting-setup.md]: we can always go through and move them to the bottom later tho!

Comment (clarkbw) [inline docs/getting-setup.md]: Ok, I get it. Sounds good.

**Pull request merged**